### PR TITLE
CTCTRADERS-2615: Initial page creation

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

File: migrations/applied_migrations/ViewAllArrivals.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (ericohwotu) [inline migrations/applied_migrations/ViewAllArrivals.sh:1]: can this be deleted?

Comment (MatthewAlexanderBeck) [reply migrations/applied_migrations/ViewAllArrivals.sh]: Yes and done

File: conf/messages.en [hunk 2]
```diff
@@ -278,3 +278,6 @@ unauthorisedWithoutGroupAccess.title = You cannot use this service group authori
 unauthorisedWithoutGroupAccess.heading = You cannot use this service
 unauthorisedWithoutGroupAccess.requestAccess = request access to NCTS
 unauthorisedWithoutGroupAccess.beforeUseThisService =  before you can use this service.
+
+viewAllArrivals.title = viewAllArrivals
+viewAllArrivals.heading = viewAllArrivals
```
Comment (amaalali) [inline conf/messages.en:283]: Do we need these assets or will we be reusing the message from `viewArrivals`?

**Review by amaalali — APPROVED** [review]

File: conf/messages.en (hunk 2 above)
Comment (MatthewAlexanderBeck) [reply conf/messages.en:283]: Ah good spot, will reuse the existing ones so I will remove these

**Pull request merged**